app/screens/settings/display_timezone_select/index.tsx

```diff
@@ -131,10 +138,9 @@ const SelectTimezones = ({selectedTimezone, onBack}: SelectTimezonesProps) => {
                 keyExtractor={keyExtractor}
                 keyboardDismissMode='on-drag'
                 keyboardShouldPersistTaps='always'
-                maxToRenderPerBatch={15}
                 removeClippedSubviews={true}
                 renderItem={renderItem}
-                viewabilityConfig={VIEWABILITY_CONFIG}
+                contentContainerStyle={{flexGrow: 1}}
```

Comment:
Not blocking, but better to have it in the styles.

Reply:
please make the change.